src/views/InternalApp/ThePlans/components/PlanForm.vue

```diff
@@ -18,10 +18,15 @@ const props = defineProps({
     type: String,
     required: true,
     default: FormTypeEnum
-  }
+  },
+  isFileValid: Boolean
```

Comment:
Is this the format on props?
https://app.clickup.com/14221792/v/dc/dj0f0-2765/dj0f0-225